### PR TITLE
Add DATEV

### PR DESCRIPTION
**Pull request by timreibe**

### CODE REVIEW

File: entries/d/datev.de.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "DATEV": {
+    "domain": "datev.de",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://apps.datev.de/help-center/documents/1022936",
```
Comment (Carlgo11) [inline entries/d/datev.de.json]: Would this be a correct description?
```suggestion
    "documentation": "https://apps.datev.de/help-center/documents/1022936",
    "notes": "2FA only available for DATEV Arbeitnehmer online",
```

**Comment by Carlgo11** [comment]
As the requested changes haven't been resolved yet I'm going to label this PR as inactive.
:warning: Unless there's activity within the next 7 days I'll close the PR.

File: entries/d/datev.de.json (hunk 1 above)
Comment (phallobst) [inline entries/d/datev.de.json]: There are other methods available as well:
```suggestion
      "totp",
      "sms",
      "custom-hardware"
    ],
    "custom-hardware": [
      "National identity card"
    ],
    "documentation": "https://apps.datev.de/help-center/documents/1022726",
    "notes": "2FA only available for DATEV Arbeitnehmer online",
```

**Review by phallobst — DISMISSED** [review]

**Comment by timreibe** [comment]
thanks @Carlgo11 for editing, the changes were correct! Is it ready to merge now?

**Review by Carlgo11 — APPROVED** [review]
Thank you! 🙂
Your contribution will be visible in around 10 minutes. Sometimes it takes longer due to server and browser cache.

**Pull request merged**